### PR TITLE
Fixes VM name conflict checking with WSL

### PR DESCRIPTION
**Pull request by n1hility**
- Corrects VM name conflict check with WSL
- Changes GetHyperVisorVMs to Exists to better abstract desired contract

```release-note
none
```

### CODE REVIEW

File: pkg/machine/e2e/init_windows_test.go [hunk 1]
```diff
@@ -75,4 +77,50 @@ var _ = Describe("podman machine init - windows only", func() {
 		Expect(session).To(Exit(125))
 		Expect(session.errorToString()).To(ContainSubstring("already exists on hypervisor"))
 	})
+
+	It("init should not should not overwrite existing HyperV vms", func() {
+		skipIfNotVmtype(define.WSLVirt, "WSL test only")
```
Comment (baude) [inline pkg/machine/e2e/init_windows_test.go:82]: s/hyperv/wsl

**Comment by baude** [comment]
LGTM, one nit on the test title ...

File: pkg/machine/e2e/init_windows_test.go [hunk 2]
```diff
@@ -75,4 +77,50 @@ var _ = Describe("podman machine init - windows only", func() {
 		Expect(session).To(Exit(125))
 		Expect(session.errorToString()).To(ContainSubstring("already exists on hypervisor"))
 	})
+
+	It("init should not should not overwrite existing HyperV vms", func() {
```
Comment (rhatdan) [inline pkg/machine/e2e/init_windows_test.go:81]: Tough sentence to parse, or were you going for emphasis on the "should not should not" :^)

**Pull request merged**